tests/v2/validation/rbac/rbac_test.go

```diff
@@ -335,6 +338,45 @@ func (rb *RBTestSuite) ValidateAddCMAsProjectOwner() {
 
 }
 
+func (rb *RBTestSuite) ValidateCreateClusterAsResAdmin() {
+	nodeRoles1 := []string{
+		"--etcd",
+		"--controlplane",
+		"--worker",
+	}
+
+	clustersConfig := new(provisioning.Config)
+	config.LoadConfig(provisioning.ConfigurationFileKey, clustersConfig)
+
+	kubernetesVersion := clustersConfig.RKE1KubernetesVersions[0]
+	cni := clustersConfig.CNIs[0]
+	nodeProviders := clustersConfig.NodeProviders[0]
+
+	externalNodeProvider := provisioning.ExternalNodeProviderSetup(nodeProviders)
+	rke1.TestProvisioningRKE1CustomCluster(rb.T(), rb.standardUserClient, externalNodeProvider, nodeRoles1, kubernetesVersion, cni)
+}
+
+func (rb *RBTestSuite) ValidateListGlobalSettings() {
+	adminListSettings, err := listGlobalSettings(rb.steveAdminClient)
+	require.NoError(rb.T(), err)
+	resAdminListSettings, err := listGlobalSettings(rb.steveStdUserclient)
```

Comment:
nit; can we have this be on a new line for "breathing room"?

Reply:
Since these are the settings for admin and restricted admin, I added them together without adding an additional line.